h2integrate/converters/hydrogen/pem_model/run_PEM_master.py

```diff
@@ -99,13 +96,10 @@ def run_grid_connected_pem(self, system_size_mw, hydrogen_production_capacity_re
         # h2_df_tot = pd.DataFrame(h2_tot, index=list(h2_tot.keys()), columns=['Cluster #0'])
         return pd.DataFrame(h2_df_ts), pd.DataFrame(h2_df_tot)
 
-    def run(self, optimize=False):
+    def run(self):
         # TODO: add control type as input!
         clusters = self.create_clusters()  # initialize clusters
-        if optimize:
```

Comment:
Why remove this? It seems like it could still be used

Reply:
It was not tested or used in any code in the repo; unclear if it works. We could keep it but I'd want to add a test. @elenya-grant could you please weigh in how you want to handle or include different pem control types here?